core/genesis.go

```diff
@@ -387,6 +387,11 @@ func SetupGenesisBlockWithOverride(db ethdb.Database, triedb *trie.Database, gen
 	if compatErr != nil && ((head.Number.Uint64() != 0 && compatErr.RewindToBlock != 0) || (head.Time != 0 && compatErr.RewindToTime != 0)) {
 		return newcfg, stored, compatErr
 	}
+	// <specular modification>
+	if (newcfg.IsLondon(head.Number) && newcfg.L2BaseFeeRecipient == common.Address{}) {
```

Comment:
I don't think we need to check here. if recipient is not set we just burn the fee.

Reply:
just want to reduce the diff here